tools/webpack-core/src/test/TestUtils.ts

```diff
@@ -33,7 +33,7 @@ export function getTestConfig(srcFile: string, pluginsToTest: any[], externalsTo
     },
     plugins: pluginsToTest,
     externals: externalsToTest,
-    optimization: { minimize: false, runtimeChunk: true },
+    optimization: { minimize: false, runtimeChunk: true, moduleIds: "named" },
```

Comment:
@pmconne this should prevent from running into this flaky snapshot test failures